packages/mobile/src/account/saga.ts

```diff
@@ -173,6 +250,11 @@ export function* watchFetchFinclusiveKYC() {
   yield takeLeading(Actions.FETCH_FINCLUSIVE_KYC, fetchFinclusiveKyc)
 }
 
+export function* watchSignedMessage() {
+  yield take(Actions.SAVE_SIGNED_MESSAGE)
+  yield call(handleUpdateAccountRegistration)
+}
```

Comment:
I wonder if we should avoid manually calling `handleUpdateAccountRegistration` from different places and instead listening to action/state changes (`while (true) { ... }`)to decide when the logic should be called.

Now that I say this, it may not be feasible because of the way `fcmToken` is retrieved though. So nevermind 😄